src/utils/string.cljs

```diff
@@ -59,3 +60,12 @@
          (take n)
          (map (comp string/upper-case str first))
          string/join)))
+
+(defn contains-emoji?
+  [s]
+  (let [regex (js/RegExp. "(\\p{Emoji_Presentation}|\\p{Extended_Pictographic})" "gu")]
+    (oops/ocall regex "test" s)))
```

Comment:
Regex to check for an emoji, I tried to create it using Clojure (i.e. `#"my-regex"`) but it throws an exception due to the syntax. So it's been created in JS, but it behaves as any Clojure regex :+1:

Reply:
Nice 🙌 . It's an interesting approach to get the regex working with Clojure because regex provides `Emoji_Presentation` and `Extended_Pictographic` out of the box.